docs/developers/queries/staging/user-answers.md

```diff
@@ -3,8 +3,8 @@ This query allows you to retrieve the details of answers made to a post's poll'.
 
 **CLI**
  ```bash
-desmos query posts poll-answers [id]
+desmos query posts user-answers [id]
```

Comment:
We should add the documentation and example with a specific user address as well here